pkg/deploy/rp.go

```diff
@@ -866,6 +877,12 @@ func (g *generator) template() *arm.Template {
 			typ = "securestring"
 		}
 		t.Parameters[param] = &arm.TemplateParameter{Type: typ}
+		if param == "keyvaultPrefix" {
+			t.Parameters[param] = &arm.TemplateParameter{
+				Type:      typ,
+				MaxLength: 24 - int(math.Max(float64(len(KvClusterSuffix)), float64(len(KvServiceSuffix)))),
```

Comment:
`int(math.Max(float64` is a bit of an anti-pattern, hard-code or define a helper function:
```
func max(a, b int) int {
	if a > b {
		return a
	}

	return b
}
```

Reply:
yea, I was doubting when I wrote this code 😄
I'll change it.